src/cpuinfo_x86.c

```diff
@@ -102,6 +102,9 @@ static void ParseCpuId(const uint32_t max_cpuid_leaf, X86Info* info) {
   features->bmi2 = IsBitSet(leaf_7.ebx, 8);
   features->vpclmulqdq = IsBitSet(leaf_7.ecx, 10);
 
+  features->smx = IsBitSet(leaf_1.ecx, 6);
+  features->sgx = IsBitSet(leaf_7.ebx, 2);
```

Comment:
Do you mind moving these a few lines above?
`sgx` before `bmi1` and `smx` before `aes`